### ISSUE TITLE
IDE crash

### ISSUE THREAD

**Issue by fantaisie-software** [fantaisie-software/purebasic#90]
Seems like we have an instability since 5.72 as reported here: https://www.purebasic.fr/english/viewtopic.php?f=4&t=75009 . Does any of you have got the issue as well ? Any idea of the commit which could be the culprit ? I checked but found nothing really concluding. If we don't have a clue, I will may be have to build several version to isolate the commit.

**Comment by tajmone** [comment]
I've read the bug report but I wasn't able to reproduce the problem. I've been using the latest PB release on Win 10 x64 and never stumbled in the problem described by some users.

I only experience some problems with the Integrated Debugger (which I reported).

Since the bug described in the Forum seems to relate to window minimizing operations, it sounds like it might be a problem related to DPI awareness (and, possibly, only affecting specific devices, or users working with more monitors?).

If I remember correctly, there was a commit related to DPI awareness in the IDE.

**Comment by tajmone** [comment]
I've tried to track down changes to IDE sources related to DPI awareness (through file history and blame) but couldn't find any significant changes (only whitespace and comments fixes).

But I do remember a discussion about some gadgets in the IDE not being DPI aware ... but maybe it was on the PB Forum, not here.

Anyhow, sifting through the commits and PRs history, I didn't notice any change that could possibly related to the IDE crashes reported. So I think it might be a problem related to Windows updates and/or drivers which might be causing issues on specific machines.

But I do admit that with the last PB update I've experienced various problems with code that stopped working, which ultimately turned out to be a bug which prevents compiling code that runs without error:

https://www.purebasic.fr/english/viewtopic.php?f=4&t=75064

Since I hadn't experienced any such issues with the previous version, I though the was something wrong with the latest compiler. Chances are that if the latest IDE was compiled with the latest PBCompiler, the problem might have affected the generated binary (instead of the problem being in the IDE sources).
